airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -924,7 +924,7 @@ def read_records(
         yield from self._flat_associations(records)
 
 
-class Campaigns(Stream):
+class Campaigns(IncrementalStream):
```

Comment:
Is this all it takes to support incrementals?

Reply:
Same for other streams, unless the endpoint supports it, we shouldn't assume that it can be don3e